src/screens/TopicMemberScreen/index.js

```diff
@@ -208,4 +207,4 @@ const TopicMemberScreen = () => {
     </SafeAreaProvider>
   );
 };
-export default withInteractionsManaged(TopicMemberScreen);
+export default TopicMemberScreen;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withInteractionsManaged` HOC has been removed from the export of `TopicMemberScreen`. This could potentially lead to undesired behavior if there were any interactions that needed to be managed. If this is not the case, then this change is fine. Otherwise, consider adding it back.
```diff
- export default TopicMemberScreen;
+ export default withInteractionsManaged(TopicMemberScreen);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->